test/semantics/modfile01.f90

```diff
@@ -27,11 +27,11 @@ module m
 
 !Expect: m.mod
 !module m
-!integer::i
-!integer,private::j
+!integer(4)::i
```

Comment:
Any reason just to duplicate the test (to make sure default are set correctly with -i8/-r8?  But there will be other tests for that.

Reply:
There is no -i8/-r8 option yet.